include/utility/includes.h

```diff
@@ -0,0 +1,10 @@
+  
+#ifndef __RF24_INCLUDES_H__
+#define __RF24_INCLUDES_H__
+
```

Comment:
I removed the nRF24 library so that whoever needs gorf24 does not need to build also nRF24 and c wrapper. Building those libraries requires custom arm compilers tailored for RPi zero, which not everyone has. I would go one step further and remove also the wrapper from gorf24 and move it into a separate repository. And I would ship only librf24.so and librf24_c.so files in gorf24.